### ISSUE TITLE
Error while requiring

### ISSUE THREAD

**Issue by SevereOverfl0w** [camsaul/methodical#30]
```
Syntax error (ExceptionInfo) compiling at (potemkin.clj:5:67).                                                                                                       
`potemkin.collections/reify-map-type` does not exist   
```

Followed by subsequent failures from:

```
#object[clojure.lang.Compiler$CompilerException "0xce492f8" "Syntax error compiling at (methodical/core.clj:1:1)."]
```

I'm using deps.edn, and here's the resolved tree:

```
methodical/methodical 0.9.4-alpha
  potemkin/potemkin 0.4.5
    clj-tuple/clj-tuple 0.2.2
  pretty/pretty 1.0.0
```

**Comment by camsaul** [comment]
Not sure why that would be happening. What version of Clojure are you using?

**Comment by SevereOverfl0w** [comment]
Trying this today gave me a different exception:

```
❯ replme methodical
Found deps {:deps {methodical/methodical {:mvn/version "0.9.4-alpha"}}}
[Rebel readline] Type :repl/help for online help info
user=> (require 'methodical.core)
Syntax error (ClassNotFoundException) compiling at (collections.clj:28:1).
clojure.core$seq_QMARK___5406
user=> *clojure-version*
{:major 1, :minor 10, :incremental 0, :qualifier nil}
user=> (clojure.repl/pst *e)
Note: The following stack trace applies to the reader or compiler, your code was not executed.
CompilerException Syntax error compiling at (potemkin/collections.clj:28:1). #:clojure.error{:phase :compile-syntax-check, :line 28, :column 1, :source "potemkin/collections.clj"}
	clojure.lang.Compiler.load (Compiler.java:7647)
	clojure.lang.RT.loadResourceScript (RT.java:381)
	clojure.lang.RT.loadResourceScript (RT.java:372)
	clojure.lang.RT.load (RT.java:463)
	clojure.lang.RT.load (RT.java:428)
	clojure.core/load/fn--6824 (core.clj:6126)
	clojure.core/load (core.clj:6125)
	clojure.core/load (core.clj:6109)
	clojure.core/load-one (core.clj:5908)
	clojure.core/load-one (core.clj:5903)
	clojure.core/load-lib/fn--6765 (core.clj:5948)
	clojure.core/load-lib (core.clj:5947)
Caused by:
NoClassDefFoundError clojure/core$seq_QMARK___5406
	potemkin.walk/walk (walk.clj:8)
	potemkin.walk/postwalk (walk.clj:20)
	potemkin.macros/unify-gensyms (macros.clj:29)
	potemkin.macros/unify-gensyms (macros.clj:29)
	potemkin.collections/eval6266 (collections.clj:29)
	potemkin.collections/eval6266 (collections.clj:28)
	clojure.lang.Compiler.eval (Compiler.java:7176)
	clojure.lang.Compiler.load (Compiler.java:7635)
	clojure.lang.RT.loadResourceScript (RT.java:381)
Caused by:
ClassNotFoundException clojure.core$seq_QMARK___5406
	java.net.URLClassLoader.findClass (URLClassLoader.java:382)
	java.lang.ClassLoader.loadClass (ClassLoader.java:424)
	sun.misc.Launcher$AppClassLoader.loadClass (Launcher.java:349)
	java.lang.ClassLoader.loadClass (ClassLoader.java:357)
	potemkin.walk/walk (walk.clj:8)
	potemkin.walk/postwalk (walk.clj:20)
nil
user=> (require 'methodical.core)
Syntax error (ExceptionInfo) compiling at (potemkin.clj:5:67).
`potemkin.collections/reify-map-type` does not exist
```

**Comment by camsaul** [comment]
Related: #33

**Comment by camsaul** [comment]
Should have been fixed by #44
